python-sdk/tests/test_bandit_result.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+# In rust implementation, `eppo_client.bandit.BanditResult` is an alias
+# to `eppo_client.EvaluationResult`.
+from eppo_client.bandit import BanditResult
+
+
+def test_bandit_result_to_string_variation():
+    result = BanditResult(variation="variation", action=None)
+    assert result.to_string() == "variation"
+
+
+def test_bandit_result_to_string_action():
+    result = BanditResult(variation="variation", action="action")
+    assert result.to_string() == "action"
+
+
+# Native python implementation does not convert variation to string.
```

Comment:
Interesting.

But yeah in the Eppo application UI, we only allow creating string-valued bandit flags

Reply:
4.0 extends `BanditResult` to `EvaluationResult` so different-typed variations are allowed. `BanditResult` is now an alias to new `EvaluationResult`

Something like this:
```python
# old BanditResult:
class BanditResult:
    variation: str
    action: str | None

# new EvaluationResult:
class EvaluationResult:
    variation: Any
    action: str | None
    evaluation_details: Any | None
```